book/src/api-reference/jsonrpc-api.md

```diff
@@ -78,6 +78,35 @@ Requests can be sent in batches by sending an array of JSON-RPC request objects
 * Signature: An Ed25519 signature of a chunk of data.
 * Transaction: A Solana instruction signed by a client key-pair.
 
+## Configuring State Confidence
+
+Solana nodes choose which bank state to query based on confidence requirements
+set by the client. Clients may specify two parameters:
+* confirmations - how many confirmed blocks the node should require
+* percentage - how much of the cluster stake must have registered `confirmations`, as a decimal between 0.0 and 1.0
+
+The confidence parameter should be included as the last element in the `params` array:
+
+```bash
+curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0", "id":1, "method":"getBalance", "params":["83astBRguLMdt2h5U1Tpdq5tjFoJ6noeGwaY3mDLVcri",{"confirmations":15,"percentage":0.60}]}' 192.168.1.88:8899
+```
+
+For convenience, the confidence configuration can include a `bankType` of either
```

Comment:
nit 1: `recent` to me doesn't add much value, I'd rather say `confirmations: 1` to avoid the ambiguity over what "recent" means (ie, 2 or 3 are pretty _recent_ too)

nit 2: `max` instead of `rooted`?   `max` feels more user friendly and obvious in meaning, whereas `rooted` is some weird Solana terminology